Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedralCorotationalFEMForceField.inl

```diff
@@ -103,6 +103,12 @@ void TetrahedralCorotationalFEMForceField<DataTypes>::init()
         this->d_componentState.setValue(sofa::core::objectmodel::ComponentState::Invalid);
         return;
     }
+
+    if (m_topology->getNbTetrahedra() == 0)
+    {
+        msg_warning() << "No tetrahedra found in linked Topology.";
```

Comment:
If there is no tetra, maybe the component should be set to Invalid ?

Reply:
No because with topological changes we are able to handle the scenario: creating an empty object and populating it at runtime.
But keeping a warning is useful for users as 99% of the case you want a valid topology

Reply:
ok